File: src/isEven.js
// Test.assertEquals(isEven(2), true)
// Test.assertEquals(isEven(3), false)
// Test.assertEquals(isEven(10), true)
// Test.assertEquals(isEven(31), false)
// Test.assertEquals(isEven(666), true)
// Test.assertEquals(isEven(777), false)
// Test.assertEquals(isEven(3482034), true)
// Test.assertEquals(isEven(3482035), false)
/**
 * @copyright https://edabit.com/challenge/nEdLGbAZQ5LaiumP6
 * @param {number} num
 * @returns
 */
export default function isEven(num) {
  return num % 2 === 0;
}

File: src/is_even.rs
pub fn is_even(num: i32) -> bool {
    num % 2 == 0
}
#[cfg(test)]
mod tests {
    use super::is_even;
    #[test]
    fn test_is_even() {
        assert_eq!(is_even(2), true);
        assert_eq!(is_even(3), false);
        assert_eq!(is_even(10), true);
        assert_eq!(is_even(31), false);
        assert_eq!(is_even(666), true);
        assert_eq!(is_even(777), false);
        assert_eq!(is_even(3482034), true);
        assert_eq!(is_even(3482035), false);
    }
}
